config/checker/com.intellij.astub

```diff
@@ -268,14 +273,7 @@ class CollectionComboBoxModel<T> {
 @UIPackage package com.intellij.ui.roots;
 @UIPackage package com.intellij.ui.speedSearch;
 @UIPackage package com.intellij.ui.switcher;
-
 @UIPackage package com.intellij.ui.table;
-
-class JBTable {
-  void updateUI(@UnknownInitialization JBTable this);
```

Comment:
See below, I'd leave this annotation even if we're not going to invoke/override this method